### DISCUSSION TITLE
OCR-ing Table of contents?

### DISCUSSION THREAD

**Discussion by maribox** [ocrmypdf/OCRmyPDF#1310] [Q&A]
Hey, 
would it theoretically be possible to OCR a table of contents? I did not find an issue tracker for this, but is this somehow planned or is it just too difficult?
Greetings

**Comment by jbarlow83** [comment]
If by that, you mean detecting that a particular range of pages are the TOC, and converting each entry to a link and constructing a PDF table of contents (outlines) object based on that? That would be quite difficult. Not impossible, but difficult.

Suppose, as a more general issue, that the TOC's page references don't align with PDF page numbers because we have a scanned document with some blank pages; or the TOC is an appendix and page numbers are relative to the beginning of the appendix....
